qemu/patches/virtiofsd/0002-add-no-pivot-root.patch

```diff
@@ -0,0 +1,91 @@
+diff -pruN nemu.old/configure nemu.new/configure
```

Comment:
thanks for the patch, but  itwould be nice to have it signed-off and with a commit message

Reply:
I have added signed-off and commit message to the patch.